src/lib/functions/registry.ts

```diff
@@ -168,25 +168,11 @@ export class FunctionsRegistry {
   }
 
   /**
-   * Runs before `scan` and calls any `onDirectoryScan` hooks defined by the
-   * runtime before the directory is read. This gives runtime the opportunity
-   * to run additional logic when a directory is scanned.
+   * TODO(serhalp): Is this really the right place to do this mutation? Should
+   * we do this at all?
```

Comment:
I can't suggest the diff but we should convert this from a jsdoc comment to an inline comment